src/style.css

```diff
@@ -0,0 +1,139 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+/*region Normalize*/
+h2,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #ccc;
+}
+/*endregion*/
+
+/*region Card*/
+.card {
+  box-sizing: border-box;
```

Comment:
move `box-sizing` property to `*` selector

Reply:
+